src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -486,6 +486,18 @@ public ResponseEntity<Mono<Void>> applyGroovyScript(@PathVariable("studyUuid") U
         return ResponseEntity.ok().body(studyService.assertCanModifyNode(nodeUuid).then(studyService.assertComputationNotRunning(nodeUuid)).then(studyService.applyGroovyScript(studyUuid, groovyScript, nodeUuid).then()));
     }
 
+    @PutMapping(value = "/studies/{studyUuid}/nodes/{nodeUuid}/network-modification/{modificationId}")
```

Comment:
We don't want something more explicit for the api route ?
For example @PutMapping(value = "/studies/{studyUuid}/nodes/{nodeUuid}/network-modification/reorder/{modificationId}")

Reply:
Or maybe the before parameter is required and its value is null when we want to set the modification to the end ?